packages/worker/src/api/controllers/global/configs.ts

```diff
@@ -518,7 +525,11 @@ export async function destroy(ctx: UserCtx) {
     await db.remove(id, rev)
     await cache.destroy(cache.CacheKey.CHECKLIST)
     if (id === configs.generateConfigID(ConfigType.AI)) {
-      await pro.quotas.removeCustomAIConfig()
+      await pro.quotas.set(
```

Comment:
Just to double check, this is not meant to use the new helper added here? https://github.com/Budibase/budibase-pro/pull/331/files#diff-a0aad30e4928dd505d83d6c01819c048f2b553140dea91a84c97ef945602d8edR32

Reply:
Nope, this particular one is to just reset all the quotas back to 0 - I could actually get rid of that helper tbh as it's not really necessary